ekf_estimator/CMakeLists.txt

```diff
@@ -1,8 +1,8 @@
 cmake_minimum_required(VERSION 3.0.2)
```

Comment:
I think ekf stuff will be merged with the robot driver package. So anything here can also be ignored?